WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderSubsActivity.java

```diff
@@ -82,6 +85,7 @@ protected void onCreate(Bundle savedInstanceState) {
         restoreState(savedInstanceState);
 
         mViewPager = (WPViewPager) findViewById(R.id.viewpager);
+        mViewPager.setOffscreenPageLimit(NUM_TABS - 1);
```

Comment:
If the tab count ever changes we won't have to worry about updating `NUM_TABS`, just one less chance for human error.